LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2020 Uber Technologies, Inc.
+Copyright (c) 2017-2020 Uber Technologies Inc.
```

Comment:
I like how Facebook handles it. They just write 2017-present and finally forget about updating this

Reply:
Good suggestion :) let me see if that's possible.